README.md

```diff
@@ -101,10 +112,11 @@ Bugsnag (https://app.bugsnag.com), clicking on settings (top right hand corner)
 select `Personal auth tokens` and then `Generate new token`.
 
 ##### BUGSNAG_ORGANIZATION
-A bugsnag organization contains all projects which errors/events will be reported on. The organization name and more 
-specifically its ID is the starting point for most Bugsnag API requests. The organization name can be obtained by 
-logging into Bugsnag (https://app.bugsnag.com), clicking on settings (top right hand corner) then `Organization settings`. 
-From here, click on the organization name and then copy the name from the pop-up window.
+A bugsnag organization contains all projects which errors/events will be reported on. The organization ID is the 
+starting point for most Bugsnag API requests. The organization ID can be obtained by opening the Chrome Devtools Network 
```

Comment:
Remove the Chrome reference. How about "by opening the Network tab in your browser's developer tools, ..."